### DISCUSSION TITLE
Unable to check the IPF versions

### DISCUSSION THREAD

**Discussion by Rofiatulumami** [alexisInSAR/EZ-InSAR#99] [Help]
Hi, excuse me i wanna ask, when i click the "Check the IPF Versions" they said error in isceprocessing. and the processing progress can't read the IPF versions until 100% it's just 33.7%. How to fix this ? thanks
<img width="422" alt="ez" src="https://github.com/alexisInSAR/EZ-InSAR/assets/161690953/a09f17e6-9235-413a-8fcd-6a04bee9c1f2">

**Comment by geoenviron98** [comment]
Maybe you need check your KML data

**Comment by alexisInSAR** [comment]
Hi, 

You can skip this step as the IPF has been updated for most Sentinel-1 acquisitions. However, it is possible that a SLC (.zip) file has not been fully downloaded. 

Kind regards, 

Alexis
